internal/chain/_token_issuance.go

```diff
@@ -12,7 +12,7 @@ import (
 
 type TokenIssuance struct{}
 
-func (TokenIssuance) Type() types.TxType { return types.TxTypeTokenCreate }
+func (TokenIssuance) Type() types.TxType { return types.TxCreateToken }
```

Comment:
I see your point. I think I should just revert everything to `types.TxType...`, because what I did is really stylistic and an unnecessary change based purely on opinion (and not engineering). @PaulSnow, WDYT?

Reply:
I reverted this change, so everything is now `TxType...`